dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -560,5 +700,32 @@ public void test_MultilangContent_IndexingAllLanguages()
         }
     }
 
+    private HTMLPageAsset createHtmlPageAsset(Language lang1, Contentlet contentlet)
+            throws DotSecurityException, WebAssetException, DotDataException {
+        final Container container = new ContainerDataGen().withContentType(contentlet
+                .getContentType(), "$!{firstName}").nextPersisted();
+
+        ContainerDataGen.publish(container);
+
+        final String uuid = UUIDGenerator.generateUuid();
+
+        final Template template = new TemplateDataGen()
+                .withContainer(container.getIdentifier(), uuid)
+                .nextPersisted();
+
+        TemplateDataGen.publish(template);
+
+        HTMLPageAsset page = new HTMLPageDataGen(folder, template).languageId(lang1.getId())
+                .nextPersisted();
+
+        HTMLPageDataGen.publish(page);
 
+        final MultiTree multiTree = new MultiTree(page.getIdentifier(),
+                container.getIdentifier(),
+                contentlet.getIdentifier(), getDotParserContainerUUID(uuid), 0);
+
+        APILocator.getMultiTreeAPI().saveMultiTree(multiTree);
+
+        return page;
+    }
```

Comment:
do we have this scenario?

Content in default language only referenced from a page in default language, and having DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE=false.

Reply:
and this one?

Content in default language only referenced from a page in default language, and having DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE=false.

Reply:
New cases added